pom.xml

```diff
@@ -33,6 +33,12 @@
                         </goals>
                         <configuration>
                             <minimizeJar>true</minimizeJar>
+                            <relocations>
```

Comment:
Why this?

Reply:
bStats requires that it is moved from its default package into a separate one. Otherwise, it throws an exception onEnable().